src/main/java/sg/ncl/MainController.java

```diff
@@ -3274,10 +3275,12 @@ public String adminUsersManagement(Model model, HttpSession session) {
 
     @RequestMapping("/admin/usage")
     public String adminTeamUsage(Model model,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 26 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)